sdk/src/main/java/com/deploygate/sdk/SdkDeviceStatesCollector.java

```diff
@@ -0,0 +1,60 @@
+package com.deploygate.sdk;
+
+import android.os.Build;
+import android.util.Pair;
+
+import com.deploygate.sdk.internal.Logger;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+final class SdkDeviceStatesCollector {
+
+    private final JSONObject states = new JSONObject();
+
+    public String getJSONString() {
+        return states.toString();
+    }
+
+    public void collectLocale() {
+        Locale defaultLocale = Locale.getDefault();
```

Comment:
We need to use AppCompat 1.6.0~ if we collect per-app language preferences with backward compatibility.

But we should avoid adding new dependencies, so I use `Locale` instead.